tsconfig.json

```diff
@@ -10,7 +10,7 @@
       "forceConsistentCasingInFileNames": true,
       "noEmit": true,
       "esModuleInterop": true,
-      "module": "esnext",
+      "module": "CommonJS",
```

Comment:
use `node` here. While we use CommonJS in the backend for Node.js, `node` has some extensions for it.

Reply:
I think I have found some insights for this specific issue : https://github.com/lukeed/uvu/issues/22

Reply:
Should be fine now. I specified in `tsconfig.json` if `ts-node` (`uvu`) is the one running the app (for tests) it should use `CommonJS` without altering the original `module` field

```json
"ts-node": {
        "transpileOnly": true,
        "compilerOptions": {
          "module": "CommonJS"
        },
        "include": [
          "tests/**/*"
        ]
 },
```